docker-compose.yml

```diff
@@ -22,17 +40,15 @@
     build:
       context: .
       dockerfile: Dockerfile.mysql
-    container_name: my-sql
+    container_name: mysql
     ports:
       - "3306:3306"
     volumes:
       - todoapp-db-data:/var/lib/mysql
     environment:
-      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_ROOT_PASSWORD=99887766aA
       - MYSQL_DATABASE=app_db
       - MYSQL_USER=app_user
-      - MYSQL_PASSWORD=1234
+      - MYSQL_PASSWORD=0987654321aA
```

Comment:
The environment variable `MYSQL_PASSWORD` is set to a specific value. It's important to avoid hardcoding sensitive information like passwords. Consider using Docker secrets or environment variables set at runtime to manage sensitive data securely.